addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -300,11 +295,10 @@ export function createRelatedModels(modelDefs, modelClasses = {}, indexes = {})
         }
 
         const Model = modelClasses[model] || Base;
-        const record = reactive(
-            new Model({ models, records, model: models[model], raw: baseData[model] })
-        );
+        const record = reactive(new Model({ models, records, model: models[model] }));
         const id = vals["id"];
         record.id = id;
+        record._raw = baseData[model][id];
```

Comment:
you could change this to 
```suggestion
        record.raw = baseData[model][id];
```
and then you would not have to make any other changes in the code, right? Simply remove the getters in this file

Reply:
Cannot remove in case of override

Reply:
but there would be no change in the `api` right?

Reply:
But overrides will crashs

Reply:
What do you mean? 
For ex `this.pos.config.raw.l10n_es_simplified_invoice_journal_id` now calls the getter, but it could simply call the property `raw` of the `config` object and the code would be the same. 

The name `_raw` of the property suggests that it is a private property, so calling it from outside is strange. It should then simply be called `raw`. This way the only diff that you need in this pr will be in the `related_models.js` file